docs/uuid.md

```diff
@@ -56,4 +56,6 @@ docker run --rm -v /path/to/course:/course prairielearn/prairielearn python /Pra
 
 ## Bulk replacement of UUIDs
 
-**Danger warning:** To change all UUIDs in a course, you can use the above `generate_uuids.py` script with the `--new` argument. **This can delete your entire course.**
+!!! danger
```

Comment:
@mwest1066 should we just remove this entirely? I think this is much less useful now that we scope UUID uniqueness at the course level.

Reply:
Removed, can add back if needed.

Reply:
This was a question for Matt; let's not resolve this just yet.